WebService/ComparisonEngineUI/ComparisonEngineUI/Constants.cs

```diff
@@ -6,8 +6,8 @@ namespace ComparisonEngineUI
 {
     class Constants
     {
-        public static string BarsUrl = "https://localhost:5001/api/Bars";
-        public static string DrinksUrl = "https://localhost:5001/api/Drinks";
-        public static string SpecificPricesUrl = "https://localhost:5001/api/SpecificPrices";
+        public static string BarsUrl = "http://localhost:8081/api/Bars/";
+        public static string DrinksUrl = "http://localhost:8081/api/Drinks/";
+        public static string SpecificPricesUrl = "http://localhost:8081/api/SpecificPrices/";
```

Comment:
are you sure these ports will be the same for everyone? since I know I tried whitelisting 8080 when we tried to make the api available on phone